live/en/getting-here.html

```diff
@@ -182,20 +182,20 @@ <h3 id="car">By car</h3>
           <div class="col-md-6 text-justify">
 
             <p>To get to the conference venue <strong>from bus station</strong>, take bus number 70 towards “Most SNP” and get off at stop “Most SNP”. The ride takes about 5 minutes, depending on the traffic. <strong>You have several options to change</strong></p>
-            <ul><li>bus number 30 or 37 and get off at stop “ZOO”. The ride takes 5-7 minutes, depending on the traffic.</li></ul>
+            <ul><li>bus number 30 or 37 and get off at stop “ZOO”. The ride takes 5-7 minutes again depending on traffic.</li></ul>
             <p>or</p>
-            <ul><li>bus number 28 or 29 or tram number 4, 5, 6 or 9 and get off at stop “Botanická záhrada”. The ride takes 6-9 minutes, depending on the traffic.</li></ul>
+            <ul><li>bus number 28 or 29 or tram number 4, 5, 6 or 9 and get off at stop “Botanická záhrada”. The ride takes 6-9 minutes, subject to traffic.</li></ul>
```

Comment:
OK, I didn't get the point. I wanted to change the wording somewhat so that I would avoid repeating the same phrase. Anyway, I checked the "subject to traffic" wording with my native English-speaking friend, she said it's OK. But I don't insist on that. ;-)